src/Playwright.Tests/TracingTests.cs

```diff
@@ -249,19 +249,18 @@ await Context.Tracing.StartAsync(new()
         var (events, resources) = ParseTrace(tracePath);
         CollectionAssert.IsNotEmpty(events);
 
-        string[] actualActionApiNames = GetActions(events);
-        string[] expectedActionApiNames = new string[] {
-                "BrowserContext.NewPageAsync",
-                "Page.GotoAsync",
-                "Page.SetContentAsync",
-                "BrowserContext.RunAndWaitForPageAsync",
-                "Page.ClickAsync",
-                "Page.EvaluateAsync",
-                "Page.RouteAsync",
-                "Page.GotoAsync",
-                "Page.GotoAsync"
+        string[] actualActionTitles = GetActions(events);
+        string[] expectedActionTitles = new string[] {
+                "BrowserContext.newPage",
+                "Frame.goto",
+                "Frame.setContent",
+                "BrowserContext.waitForEventInfo",
```

Comment:
not a strong fan of this waitForEventInfo here, since its internal! It should stay RunAndWaitForPageAsync or something related to that.